plugins/sudo/sudo.plugin.sh

```diff
@@ -1,11 +1,21 @@
 #! bash oh-my-bash.module
 
 function _omb_plugin_sudo_add_sudo {
+  function __omb_plugin_sudo_is_inserted {
+    [[ "${READLINE_LINE::4}" = "sudo" ]]; return $?
+  }
   if [[ -z $READLINE_LINE ]]; then
     READLINE_LINE="$(fc -ln -1 | command sed 's/^[[:space:]]\{1,\}//')"
+    ((READLINE_POINT += ${#READLINE_LINE}))
```

Comment:
I believe we can just assign the length without performing the arithmetic evaluation.

```suggestion
    READLINE_POINT=${#READLINE_LINE}
```

Reply:
Ok, I was thinking that it could have some value from the shell character